packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -529,7 +529,7 @@ export default class Extension {
   }
 
   private updateCurrentTabs ({ urls }: RequestActiveTabsUrlUpdate) {
-    this.#state.udateCurrentTabsUrl(urls);
+    this.#state.updateCurrentTabsUrl(urls);
```

Comment:
should be the inverse. this function becomes the actual implementation. the misspelled function just calls this one